### ISSUE TITLE
It is redirected to Login page on every process

### ISSUE THREAD

**Issue by NeilSavargaonkar** [ProcessMaker/processmaker#3884]
Redirects to login page even after successful login and correct credentials entered.
This occurs right after installing the and setup pm4 via docker
possible issue: POST request redirects to GET request (checked via browser console)

1. install and setup PM4 via docker 
2. Go to localhost:8080
3. Enter username and password as admin / admin

**Screenshots**
![Screenshot_20210707_182213](https://user-images.githubusercontent.com/72607743/124762224-4ef52b00-df50-11eb-990a-efa38608de4e.png)
![Screenshot_20210707_182331](https://user-images.githubusercontent.com/72607743/124762428-8368e700-df50-11eb-943a-eb645fc77741.png)

Specs:
 - Docker image version -  4.1.2
 - Browser : Chrome

**Comment by EricRossC** [comment]
password admin123 ?

**Comment by NeilSavargaonkar** [comment]
> password admin123 ?

Nope.
Checked and reset the password in the database.
Default password - 'admin'

**Comment by cristianvillalba** [comment]
Hi,
I had the same issue and the problem was that the cookie was not set.
Digging around I found that the running instance was using http (not https).

There is an option in config/session.php (around line 178) that I changed and solve my problem:
...
'secure' => env('SESSION_SECURE_COOKIE', true),
...
I put secure to false and now I can login

ps: I think it will be better to change the .env file and put the SESSION_SECURE_COOKIE on false there, but I didn't test it.

Hope that your issue gets resolved.
